ixwebsocket/IXWebSocketTransport.cpp

```diff
@@ -543,7 +543,7 @@ namespace ix
             }
 
             // Prevent integer overflow in the next conditional
-            const uint64_t maxFrameSize(1 << 63);
+            const uint64_t maxFrameSize(uint64_t(1) << 63);
```

Comment:
Can you make this 1LL << 63 ?

Reply:
or 1ULL or something like that, see if there's an option not to do the full cast to uint64_t ?